src/screen/EditGifScreen.jsx

```diff
@@ -0,0 +1,340 @@
+    color: "",
+    grid: "",
+    fps: "",
+  });
+
+  const showToastMessage = () => {
+    ToastAndroid.showWithGravity(
+      "저장 완료",
+      ToastAndroid.CENTER,
+      ToastAndroid.SHORT,
+    );
+  };
+
+  const convertVideoToGif = async () => {
+    setIsLoading(true);
+
+    try {
+      const response = await fetch(
+        `${process.env.API_SERVER_URL}/api/videos/gif`,
+        {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+            Authorization: `Bearer ${idToken}`,
+          },
+          body: JSON.stringify({
+            videoUrl: uri,
+            filter,
+          }),
+        },
+      );
+
+      const data = await response.json();
+
+      if (data.result === "ng") {
+        console.log("실패");
+        return;
+      }
+
+      setGifUrl(data.file);
+      setIsLoading(false);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const saveGif = async () => {
+    try {
+      const filename = `${Date.now() + ".gif"}`;
+      const fileUri = `${FileSystem.documentDirectory}${filename}`;
+      const downloadedFile = await FileSystem.downloadAsync(gifUrl, fileUri);
+      const permission = await MediaLibrary.requestPermissionsAsync();
+
+      if (permission.status === "granted") {
+        setHasLibraryPermissions(permission.status === "granted");
+      }
+
+      if (permission.status !== "granted") {
+        navigation.navigate("Home");
+      }
+
+      const asset = await MediaLibrary.createAssetAsync(downloadedFile.uri);
+      const album = await MediaLibrary.getAlbumAsync("Download");
+
+      if (album === null) {
+        await MediaLibrary.createAlbumAsync("Download", asset, false);
+      } else {
+        await MediaLibrary.addAssetsToAlbumAsync([asset], album, false);
+      }
+
+      showToastMessage();
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const colorButtonPress = (title) => {
+    if (title === filter.color) {
+      setfilter({ ...filter, color: "" });
+    } else {
+      setfilter({ ...filter, color: title });
+    }
+  };
+
+  const gridButtonPress = (title) => {
+    if (title === filter.grid) {
+      setfilter({ ...filter, grid: "" });
+    } else {
+      setfilter({ ...filter, grid: title });
+    }
+  };
+
+  const fpsButtonPress = (title) => {
+    if (title === filter.fps) {
+      setfilter({ ...filter, fps: "" });
+    } else {
+      setfilter({ ...filter, fps: title });
+    }
+  };
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.header}>
+        <TouchableOpacity
+          onPress={() => navigation.goBack()}
+          style={styles.backButton}
+        >
+          <AntDesign name="left" size={27} color="black" />
+        </TouchableOpacity>
+      </View>
+      <View style={styles.contentsContainer}>
+        <Image
+          resizeMode="cover"
+          source={{
+            uri: gifUrl ? gifUrl : exampleGif,
```

Comment:
다른 목적이 없다면 exampleGif 가 필요없어 보이네요. 
```suggestion
            uri: gifUrl,
```

Reply:
현재 예시 gif을 보여주고 있기 때문에 추가하였습니다.
프론트에서 gif 적용 화면을 보여줄 수 있다면 기능에 맞추어 삭제하고자 합니다!!